modules/local/kraken2_db_preparation.nf

```diff
@@ -14,10 +14,21 @@ process KRAKEN2_DB_PREPARATION {
 
     script:
     """
-    mkdir db_tmp
-    tar -xf "${db}" -C db_tmp
-    mkdir database
-    mv `find db_tmp/ -name "*.k2d"` database/
+    if [[ -d ${db} ]]; then
```

Comment:
Theoretically we can do the checking if it's a directory with Nextflow's `.isDirectory()` function as well as checkign the name... but this can updated in the future (preferably as making symlinks of symlinks can be dodgy in some computing env contexts)